datadriven/tests/tmpsgdeconfig.json

```diff
@@ -0,0 +1 @@
+{"dataSource" : { "filePath" : "datadriven/datasets/densityEstimation/3D_KurB4B1.csv", "hasTargets" : false},"scorer" : { "metric" : "NLL"},"fitter" : { "type" : "densityEstimation", "gridConfig" : { "gridType" : "linear","level" : 5},"adaptivityConfig" : {"numRefinements" : 3, "threshold" : 0.001,"maxLevelType" : false, "noPoints" : 3},"regularizationConfig" : {"lambda" : 1}, "densityEstimationConfig" : { "densityEstimationType" : "decomposition"}}}
```

Comment:
Where is this file needed? If it is to be kept, it should be formatted so be easily readable.

Reply:
The file is indeed generated by the test cases. I removed it now.

Reply:
Then it should be removed by the test as soon as it not needed anymore. If that is not possible, we need to make sure that it will not appear `$ git status` (eg via .gitignore).

Reply:
The tests are already removing the file. I don't know why it didn't work for me once. Probably I stopped the tests at a wrong moment and commited the changes.